libs/store/src/lib/reducers/logger.ts

```diff
@@ -0,0 +1,33 @@
+import { AppErrorTimestamp, AppStateModel } from '@razor/models';
+import { logErrorReducerPayload } from '../payloads';
+
+export const logErrorReducer = (
+  state: AppStateModel,
+  payload: logErrorReducerPayload,
+): AppStateModel => {
+  const { errorLog, errorTimestamp } = payload;
+  let logModel = { ...state.errorLogsModel };
+
+  const maxLogs = 1024;
+  const logsLength = Object.keys(state.errorLogsModel).length;
+
+  if (logsLength > maxLogs - 1) {
+    const lastKey: AppErrorTimestamp = Object.keys(state.errorLogsModel)[0];
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { [lastKey]: removedLog, ...newLogModel } = state.errorLogsModel;
+    logModel = { ...newLogModel };
+  }
+
+  console.log(state.errorLogsModel);
```

Comment:
You can't do console logs within reducers. 
A reducer needs to be a pure function, without side effects.
console.log is a side effect. Hence must be done within an effect.

Reply:
This is removed in PR #14 https://github.com/the-ai-team/razor/pull/14/files#diff-65c47b8507de32a7a9165c89e550a79ba994cff94ce4489d5cbc7ba257ddcfb9